src/Features/Core/Portable/xlf/FeaturesResources.fr.xlf

```diff
@@ -22,6 +22,11 @@
         <target state="translated">Les actions ne peuvent pas être vides.</target>
         <note />
       </trans-unit>
+      <trans-unit id="Convert_to_linq">
+        <source>Convert to linq</source>
+        <target state="new">Convert to linq</target>
+        <note />
+      </trans-unit>
       <trans-unit id="Convert_to_query">
```

Comment:
we have convert_to_query and convert_to_linq.  what is the difference?

Reply:
i'd prefer `convert_to_linq_calls` myself

Reply:
alternatively, offer these:

1. Convert to LINQ
2. Convert to LINQ (call form)

Reply:
I like your suggestions, I will update.